### ISSUE TITLE
Page Speed Insights and CrUX API should not be running 'experimental' accessibility audits

### ISSUE THREAD

**Issue by brgibson** [GoogleChrome/web.dev#10417]
**Describe the bug**
Page Speed Insights and CrUX API should not be reporting 'experimental' accessibility audits as failures.

**To Reproduce**
1. Browse to https://pagespeed.web.dev/analysis/https-www-edmunds-com-ford-f-150/k33kin2gr0?utm_source=psi&utm_medium=redirect&form_factor=mobile or run a test for https://www.edmunds.com/ford/f-150/ in https://pagespeed.web.dev/
2. Scroll down to the Accessibility section of the lab results.
3. Notice that the audit "Elements with visible text labels do not have matching accessible names." is showing in the audits section.
4. Read the description of the audit and browse to the link provided https://dequeuniversity.com/rules/axe/4.7/label-content-name-mismatch
5. Notice that this is currently an experimental rule ("label-content-name-mismatch")
6. You can also see that this is listed as an experiment rule here: https://dequeuniversity.com/rules/axe/4.7/#experimental-rules
7. Notice that per dequeu "label-content-name-mismatch" is "[a rule] we are still testing and developing. They are disabled by default in axe-core, but are enabled for the axe browser extensions."

**Expected behavior**
1. Page speed insights (and corresponding CrUX api) should not be reporting on experimental accessibility rules.  

Note: PSI is currently inconsistent with both Chrome and Chrome Canary Lighthouse Accessibility Audits, which do not have this bug.

**Screenshots**
<img width="1181" alt="image" src="https://github.com/GoogleChrome/web.dev/assets/1953625/f801d53d-bf06-40e2-888c-58d1fae82dbc">

<img width="1199" alt="image" src="https://github.com/GoogleChrome/web.dev/assets/1953625/34c9566f-d03b-428b-bad6-a1b48862a6d0">

<img width="1119" alt="image" src="https://github.com/GoogleChrome/web.dev/assets/1953625/42db9afb-2afd-4d5c-bb75-6704c556ab12">

**Comment by brgibson** [comment]
I've requested help here https://groups.google.com/a/chromium.org/g/chrome-ux-report/c/w2h9VCUv0mc to try to get some traction (since I am not sure if this is the correct place to submit it).

**Comment by rviscomi** [comment]
Thanks @brgibson. Closing this issue and transferring it to the Lighthouse repo: https://github.com/GoogleChrome/lighthouse/issues/15519
